src/base/init/parser.y

```diff
@@ -1096,7 +1096,7 @@ x_flag		: L_DISPLAY string_expr	{ free(config.X_display); config.X_display = $2;
 		| X_FULLSCREEN bool   { config.X_fullscreen = $2; }
 		| X_NOCLOSE bool      { config.X_noclose = ($2!=0); }
 		| X_NORESIZE bool     { config.X_noresize = ($2!=0); }
-		| VGAEMU_MEMSIZE expression	{ config.vgaemu_memsize = $2; }
+		| VGAEMU_MEMSIZE expression	{ config.vgaemu_memsize = ($2 ? $2 : 1024); }
```

Comment:
Is this 1024 still needed?
I suppose we need to split lfb and
main vga mem?
Not in this patch-set of course.

Reply:
Converted this to config_scrub().